plugins/ltac/rewrite.mli

```diff
@@ -89,7 +89,7 @@ val add_setoid :
   rewrite_attributes -> local_binder_expr list -> constr_expr -> constr_expr -> constr_expr ->
   Id.t -> unit
 
-val add_morphism_infer : rewrite_attributes -> constr_expr -> Id.t -> unit
+val add_morphism_infer : ?ontop:Proof_global.t -> rewrite_attributes -> constr_expr -> Id.t -> Proof_global.t option
 
```

Comment:
IIUC this allows Add Morphism to be nested with other proofs (using ontop), I'm not sure it's necessary to give that capability here.

Reply:
Indeed, this is in the case you want to perform `Add Morphism` inside an open proof.